src/bFS/Node.java

```diff
@@ -0,0 +1,26 @@
+package bFS;
```

Comment:
ok so you don't need to make a extra file for the `Node` class. You can simply write a `public static class` inside the `BFS.java` file